definitions/ext-nas/golden_metrics.yml

```diff
@@ -1,16 +1,12 @@
-# Tested on Netgear ReadyNAS
 cpuUtilization:
-  title: CPU Utilization (%)
+  title: CPU
   unit: PERCENTAGE
-  query:
-    select: average(kentik.snmp.CPU)
-    from: Metric
-    where: "provider = 'kentik-nas'"
-
-memoryUtilization:
-  title: Memory Utilization (%)
-  unit: PERCENTAGE
-  query:
-    select: average(kentik.snmp.MemoryUtilization)
-    from: Metric
-    where: "provider = 'kentik-nas'"
+  queries:
+    kentik/readynas:
+      select: 100 - latest(kentik.snmp.ssCpuIdle)
+      from: Metric
+      where: "provider = 'kentik-nas' AND SysObjectID = '.1.3.6.1.4.1.4526.100.16'"
```

Comment:
They always have the same SysObjectID?

Reply:
We'll merge once confirmed!

Reply:
The Netgear ReadyNas do, that's been in the definition since day 1. We need to refactor this but we don't have any customers sending us this data to test against right now unfortunately so it's best to leave it as is IMO.